stdlib/email/utils.pyi

```diff
@@ -28,9 +28,21 @@ _PDTZ: TypeAlias = tuple[int, int, int, int, int, int, int, int, int, int | None
 
 def quote(str: str) -> str: ...
 def unquote(str: str) -> str: ...
-def parseaddr(addr: str | None) -> tuple[str, str]: ...
+
+if sys.version_info >= (3, 13):
+    def parseaddr(addr: str, *, strict: bool = True) -> tuple[str, str]: ...
```

Comment:
`None` works, since `parseaddr` is designed to handle malformed inputs, but I don't it makes sense to accept as accept for either the new or old definition.

Reply:
Seems like it also explicitly supports lists: https://github.com/python/cpython/blob/b5805892d55e769335c11a994b586355720263ba/Lib/email/utils.py#L341

Reply:
let me fix that.

Reply:
Actually - I'm not sure we want to add that. It just seems to be trying to handle the most flexible input possible, no?

Reply:
We should add back `None` in that case as well I would say.

Reply:
It does also have a thing where it returns `('', '')` for any non-str input, but this is a different check. I think we should add `list[str]` because it has specific handling for this type.

Reply:
https://github.com/python/typeshed/pull/12336/commits/135c9d0b6be3f2ebb4a0583fd0c70721c9ed23fc